app/views/onboarding/LocationsPermissions.tsx

```diff
@@ -30,13 +30,12 @@ import {
 const LocationsPermissions = (): JSX.Element => {
   const { t } = useTranslation();
   const dispatch = useDispatch();
-  const { authSubscription, location } = useContext(PermissionsContext);
```

Comment:
Not sure if this is needed?

Reply:
by this, do you mean `location`? if so i believe yes.

Reply:
Sorry, I meant to only remove the authSubscription here. Fixing!

Reply:
No, I meant the authSubscription inside the LocationsPermissions. Below, I kept the location stuff still.